admin/spam/views.py

```diff
@@ -37,47 +37,90 @@ def get_context_data(self, **kwargs):
         page_size = self.get_paginate_by(queryset)
         paginator, page, queryset, is_paginated = self.paginate_queryset(
             queryset, page_size)
-        context = {
+        return {
             'spam': map(serialize_comment, queryset),
             'page': page,
             'status': self.status,
             'page_number': page.number
         }
-        return super(SpamList, self).get_context_data(**context)
+
+
+class UserSpamList(SpamList):
+    template_name = 'users/spam.html'
+
+    def __init__(self):
+        self.user_id = None
```

Comment:
Thanks, this may well be fixed in a yet unmerged PR. I don't remember, hard to keep track of in a dozen interrelated PRs. @sloria @GaryKriebel